test/Sema/generic-subscript.swift

```diff
@@ -1,44 +1,34 @@
 // RUN: %target-typecheck-verify-swift
 
-
-protocol P
-{
-    subscript<Value>(x:Value) -> Int // expected-note {{protocol requires subscript with type '<Value> (Value) -> Int'; do you want to add a stub?}}
-    {
-        get 
-    } 
+protocol P {
+  subscript<Value>(x: Value) -> Int  // expected-note {{protocol requires subscript with type '<Value> (Value) -> Int'; do you want to add a stub?}}
+  {
+    get
+  }
 }
 
-struct S:P // expected-error {{type 'S' does not conform to protocol 'P'}}
+struct S: P  // expected-error {{type 'S' does not conform to protocol 'P'}}
```

Comment:
Not that it really matters but one space before a trailing comment in enough.

Reply:
> Thanks for taking on the issue! Line breaks before `{` are still inconsistent though; could you fix that?
> 
> Please consider using more idiomatic and descriptive commit messages. For example, `Fix test formatting after #<pull request id>`.

Got it, will do in my next PR!